server/routes/sentry/sentry_routes.py

```
@@ -0,0 +1,486 @@
+        "accessible_projects": accessible_projects,
+        "validated_at": datetime.now(timezone.utc).isoformat(),
+    }
+
+    try:
+        store_tokens_in_db(user_id, token_payload, "sentry")
+        logger.info(
+            "[SENTRY] Stored credentials for user %s (org=%s)",
+            sanitize(user_id), sanitize(org_slug),
+        )
+    except Exception as exc:
+        logger.exception("[SENTRY] Failed to store credentials: %s", exc)
```

Ruff (0.15.12)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)